<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>query</class>
 <widget class="QMainWindow" name="query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="current_account">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>当前账户*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>491</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>账户余额</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交易类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交易金额</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交易时间</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="return_to_main">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主页</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
